src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -173,8 +180,15 @@ private void setListMethodFields(
     PageStreamingConfig pageStreaming = context.getMethodConfig().getPageStreaming();
     TypeRef resourceType = pageStreaming.getResourcesField().getType();
     String resourceTypeName = context.getTypeTable().getAndSaveNicknameForElementType(resourceType);
+    String resourceFieldName =
+        context.getNamer().getFieldGetFunctionName(pageStreaming.getResourcesField());
```

Comment:
Hmm maybe we want to do a split in NameFormatter:

* methodName -> publicMethodName, privateMethodName
* varName -> publicVarName, privateVarName